packages/utils/telemetry-utils/src/test/utils.spec.ts

```diff
@@ -8,18 +8,30 @@ import {
 	ConfigTypes,
 	IConfigProviderBase,
 	ITelemetryBaseEvent,
-	ITelemetryBaseLogger,
 } from "@fluidframework/core-interfaces";
 import { IEventSampler, createSampledLogger, logIfFalse } from "../utils";
 import { TelemetryDataTag, tagCodeArtifacts, tagData } from "../logger";
 import { mixinMonitoringContext } from "../config";
 import { ITelemetryGenericEventExt, ITelemetryLoggerExt } from "../telemetryTypes";
 
-class TestLogger implements ITelemetryBaseLogger {
-	send(event: ITelemetryBaseEvent): void {
+class TestLogger implements ITelemetryLoggerExt {
```

Comment:
this should remain a ITelemetryBaseLogger

Reply:
It's actually going away with the removal of the deprecated `logIfFalse()`, only used in its unit tests.

Reply:
Resolved after rebase